tests/promptflow/test_promptflow_model_export.py

```diff
@@ -0,0 +1,75 @@
+from pathlib import Path
+
+import pytest
+
+from promptflow import load_flow
+from promptflow._sdk.entities._flow import Flow
+
+import mlflow
+from mlflow import MlflowException
+
+
+def get_promptflow_example_model():
+
+    flow_path = Path(__file__).parent / "flow_with_additional_includes"
```

Comment:
Do we have to keep the folder in order to test? Is there any functions that you could directly save a flow like this?

Reply:
Yes, we can't write a flow directly using python from scratch for now, need to keep this folder.